forge-gui/res/cardsfolder/h/honor_the_fallen.txt

```diff
@@ -1,8 +1,9 @@
 Name:Honor the Fallen
 ManaCost:1 W
 Types:Instant
-A:SP$ ChangeZoneAll | Cost$ 1 W | ChangeType$ Creature | Origin$ Graveyard | Destination$ Exile | RememberChanged$ True | ForgetOtherRemembered$ True | SubAbility$ DBGainLife | SpellDescription$ Exile all creature cards from all graveyards. You gain 1 life for each card exiled this way.
-SVar:DBGainLife:DB$ GainLife | Defined$ You | LifeAmount$ X
+A:SP$ ChangeZoneAll | Cost$ 1 W | ChangeType$ Creature | Origin$ Graveyard | Destination$ Exile | RememberChanged$ True | SubAbility$ DBGainLife | SpellDescription$ Exile all creature cards from all graveyards. You gain 1 life for each card exiled this way.
```

Comment:
Remove Cost?

Reply:
hmn ok, but a cleanup PR for all might be easier eventually

Reply:
i normally try to update files I'm going to touch anyways